### DISCUSSION TITLE
System.Exception error thrown when trying to add a second installer

### DISCUSSION THREAD

**Discussion by Frittions** [wixtoolset/issues#7924] [Questions] [answered]
Hello,

I am trying to add a second installer to my solution and am getting the following error message:

![image](https://github.com/wixtoolset/issues/assets/97322291/46971e79-fc74-4486-b592-2a2da24573eb)

My solution has the following structure:

![image](https://github.com/wixtoolset/issues/assets/97322291/3be36906-8557-4136-8375-c26febeb97d4)

My end goal is that I want to create two different MSI packages that will be used in the bootstrapper. I need to run a batch file operation between the two msi installations in the bootstrapper. Does anyone know what is causing this warning? I'm fairly new to WiX, but should I be able to add multiple installer projects to a solution? Is there a way to create multiple MSI packages from just a single installer?

My set-up is that I'm using VS2022 with Wix4 and WixToolset.Sdk 4.0.2"

**Comment by robmen** [comment, marked as answer]
That looks like a HeatWave issue. [FireGiant](https://www.firegiant.com/) provides HeatWave and non-FireGiant customers can [report issues here](https://github.com/firegiant/HeatWaveSupport/issues).
